test/check-timezone

```diff
@@ -0,0 +1,206 @@
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; If not, see <http://www.gnu.org/licenses/>.
+
+import datetime
+from zoneinfo import ZoneInfo
+
+from anacondalib import VirtInstallMachineCase
+from installer import Installer
+from language import Language
+from testlib import nondestructive, test_main
+from timezone import DateAndTime
+
+
+@nondestructive
+class TestDateAndTime(VirtInstallMachineCase):
+
+    def setUp(self):
+        super().setUp()
+        self.allow_journal_messages("Error .* data: Connection reset by peer")
+
+    def testDefaultState(self):
+        """
+        Description:
+            Test default state picked up from backend
+
+        Expected results:
+            - WebUI auto date/time and timezone matches backend
+            - NTP, timezone, date and time match backend
+        """
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+        dt = DateAndTime(b, m)
+
+        i.open()
+        i.reach(i.steps.DATE_TIME)
+        dt.check_auto_date_time(dt.dbus_get_ntp_enabled())
+        dt.check_auto_timezone(True)
+        ntp_servers = dt.dbus_get_ntp_servers()
+        if ntp_servers:
+            dt.check_ntp_server(ntp_servers[0])
+        else:
+            print("WARNING: No NTP servers returned from backend, skipping NTP check.")
+        dt.check_timezone_label(dt.dbus_get_timezone())
+
+    def testManualDateTime(self):
+        """
+        Description:
+            Test manual entry of date and time.
+
+        Expected results:
+            - After disabling auto, manual values are applied to backend
+        """
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+        lang = Language(b, m)
+        dt = DateAndTime(b, m)
+
+        i.open()
+        lang.select_locale("de_DE")
+        i.reach(i.steps.DATE_TIME)
+        dt.set_auto_date_time(False)
+        dt.set_date("2024-12-24")
+        dt.check_date("2024-12-24")
+        dt.set_time("18:30")
+        dt.check_time("18:30")
+        self.assertIn("2024-12-24T18:30", dt.dbus_get_system_datetime())
+
+    def testNtpServerSelection(self):
+        """
+        Description:
+            Test selecting another NTP server.
+
+        Expected results:
+            - Selected server is shown and set on backend
+        """
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+        dt = DateAndTime(b, m)
+
+        i.open()
+        i.reach(i.steps.DATE_TIME)
+        dt.select_ntp_server("2.fedora.pool.ntp.org")
+        dt.check_ntp_server("2.fedora.pool.ntp.org")
+
+    def testManualTimezone(self):
```

Comment:
Can we do some test merges? Aka all the manual tests together?

Keep in mind to always maintain a balance, by:
* making sure tests have a well defined purpose (description, expected results)
* the purpose is not too small, we don't want two many small tests, as the test setup overhead can increase a lot the time of the test runs.